tutorials/haskell/golden-testing-in-haskell-with-hspec-gold/code/src/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2019
```

Comment:
@MedAdrian Could you please update `name here`? I see we've been setting Stack Builders in [other posts](https://github.com/stackbuilders/tutorials/blame/tutorials/tutorials/haskell/mustache-templates/code/LICENSE.md#L3).